.github/workflows/ci.yml

```
@@ -0,0 +1,24 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        python-version: [3.9, 3.8, 3.7, 3.6]
+        os: [macos-latest, windows-latest, ubuntu-latest]
+    runs-on: ${{ matrix.os }}
+    env:
+      IS_COVERAGE_ALLOWED: ${{ secrets.CODACY_PROJECT_TOKEN != '' }}
+      IS_MASTER_BRANCH: ${{ github.ref == 'refs/heads/master' }}
+    steps:
+    - uses: actions/checkout@master
+    - uses: actions/setup-python@v1
```

Need `v2` for `-dev` Python versions.
```suggestion
    - uses: actions/setup-python@v2
```